controllers/admin/AdminStatsController.php

```diff
@@ -259,49 +259,78 @@ public static function getTotalProducts()
         );
     }
 
-    public static function getTotalSales($date_from, $date_to, $granularity = false)
+    /**
+     * @param string $date_from
+     * @param string $date_to
+     * @param bool $granularity
+     * @param bool $removeRefunds
+     *
+     * @return array|false|string
+     *
+     * @throws PrestaShopDatabaseException
+     */
+    public static function getTotalSales($date_from, $date_to, $granularity = false, $removeRefunds = false)
```

Comment:
Maybe silly question, but does this behaviour need to be optional? Or could it be the default behaviour?

I get that without this parameter it changes the behaviour and can be considered as a BC, but maybe every use cases actually need this change/fix So if all use cases need the change you can simply update the method and force to remove funds no?

It would also avoid updating the module to use this new extra parameter.

Reply:
So the question is: does any other module (or core code) will need to use:

`AdminStatsController::getTotalSales($date_from, $date_to, $granularity, true)`
AND
`AdminStatsController::getTotalSales($date_from, $date_to, $granularity, false)`

Reply:
(Also don't get me started with static function in a controller, that's another topic 😅  )

Reply:
I think it's optional because when you want sales, you may want remove refunds or not. 

For static : I agree. May be in v2 for `dashtrends`.